packages/documentation-site/patternfly-docs/patternfly-docs.source.js

```diff
@@ -128,4 +114,43 @@ module.exports = (sourceMD, sourceProps, sourceFunctionDocs) => {
 
   sourceProps(path.join(consoleSrcPath, "/**/*.tsx"), reactPropsIgnore);
   sourceMD(path.join(consoleDocsPath, "/examples/*.md"), 'react');
+
+  // Component Groups extension (Currently in PRERELEASE, so only include in PRERELEASE builds)
```

Comment:
NIT - this `if` case for the `PRERELEASE` flag is nested in the middle of a bunch of other extensions.  For organization (as this section could grow in the future), can we move this condition to the top or the bottom to group the code?

Something like

1. [always included] at the top of the source file, 
2. [if !EXTENSIONS_ONLY] in the middle, and 
3. [PRERELEASE] at the bottom?